.circleci/config.yml

```diff
@@ -0,0 +1,46 @@
+version: 2.1
+
+
+common: &common
+  working_directory: ~/project
+  steps:
+    - checkout
+    - restore_cache:
+        keys:
+          - v2-deps-{{ .Environment.CIRCLE_JOB }}-{{ checksum "requirements.txt" }}-{{ checksum "tox.ini" }}
+    - run:
+        name: install dependencies
+        command: pip install --user tox
+    - run:
+        name: run tox
+        command: ~/.local/bin/tox -i https://$PYPI_USER:$PYPI_PASS@pypi.mvpl.eu/simple/
+    - save_cache:
+        paths:
+          - .tox
+          - ~/.cache/pip
+          - ~/.local
+          - ./eggs
+        key: v2-deps-{{ .Environment.CIRCLE_JOB }}-{{ checksum "requirements.txt" }}-{{ checksum "tox.ini" }}
+
+
+jobs:
+  py27:
+    <<: *common
+    docker:
+      - image: circleci/python:2.7
+        environment:
+          TOXENV=py27
+  py38:
+    <<: *common
+    docker:
+      - image: circleci/python:3.8
+        environment:
+          TOXENV=py38
+
+
+workflows:
+  version: 2.1
+  test:
+    jobs:
+      - py27
+      - py38
```

Comment:
I'm wondering if we do need two separate jobs? Don't sure yet, but I can miss something? Shouldn't tox take care of proper Python version to be setuped?